dbeaver/dbeaver
# #36334 Fix default privileges for PostgreSQL

<img width="848" alt="image" src="https://github.com/user-attachments/assets/04aa23d2-84df-4a86-8793-55ccf9d229cb">

How to test:
1.	Create a test role.
2.	Select the Tables folder.
3.	Grant all privileges to the role.
4.	Verify the granted privileges.
5.	Revoke the privileges.
6.	Grant a few privileges.
7.	Verify the updated privileges.
8.	Create a table.
9.	Check the role’s privileges for the table.
